### PR TITLE
Sus 177/adding project endpoints

### PR DESCRIPTION
**Pull request by JiaanNie**

### CODE REVIEW

**Comment by JiaanNie** [comment]
oh i check out this branch from the access branch, hence you can see some access changes you can just ignored. there is already another PR related to all the access changes

**Review by ghost — APPROVED** [review]

File: src/services/sam/projects/projects.ts [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+import {AxiosInstance, AxiosResponse} from "axios";
+import {
+  GetMemberProjectsResponse,
+  CreateProjectResponse,
+  GetProjectResponse,
+  UpdateProjectResponse,
+  GetProjectMembersResponse,
+  CreateProjectKeyResponse,
+  RemoveProjectKeyResponse,
+  AddPermissionToKeyResponse, RemovePermissionFromKeyResponse, GetPermissionFromKeyResponse
```
Comment (ShawnClake) [inline src/services/sam/projects/projects.ts]: Any reason why these 3 don't get their own lines?

File: src/services/sam/projects/projects.ts [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+    this.axiosClient = axiosClient;
+    this.logger = logger;
+  }
+
+  getMemberProjects(memberUuid: string, role: string): Promise<Array<GetMemberProjectsResponse>> {
+    const params = {
+      role: role
+    }
+    return this.axiosClient.get<Array<GetMemberProjectsResponse>>(
+      `/sam/projects/member/${memberUuid}/`,
```
Comment (ShawnClake) [inline src/services/sam/projects/projects.ts]: There's a trailing slash on this endpoint. This can cause bugs/issues.

**Review by ShawnClake — CHANGES_REQUESTED** [review]
Looks good other than two minor issues. Once fixed, go ahead and merge!

File: src/services/sam/projects/projects.ts (hunk 1 above)
Comment (JiaanNie) [reply src/services/sam/projects/projects.ts]: nope ctrl alt L didnt work for some reason i can fix that

**Pull request merged**